gtest-parallel

```diff
@@ -82,6 +87,45 @@ class RawFormat:
   def end(self):
     pass
 
+# Record of test runtimes. Has built-in locking.
+class TestTimes(object):
+  def __init__(self, save_file):
+    "Create new object seeded with saved test times from the given file."
+    self.__old_times = {}  # (test binary, test name) -> runtime in seconds
```

Comment:
I think you're right regarding old_times and new_times. I'll combine them.

Prefixing with __ isn't about style, it's how member variables are made private in Python. See e.g. https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references.